src/components/testCaseList/TruncateText.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Button from "@mui/material/Button";
+import Tooltip from "@mui/material/Tooltip";
+import { truncateInput } from "../../util/Utils";
+
+const TruncateText = ({ text, maxLength = 60, name, dataTestId }) => {
+  if (text && text.length > maxLength) {
+    const displayText = truncateInput(text, maxLength);
+    return (
+      <td>
```

Comment:
both here and down in `else` part, I'm not sure about adding `td` element. This might add `td` inside `td` becauae you have `StyledCell` wrapping this component which already has a `td`. Also, we may not be able to reuse this component anywhere apart from just a table. Rest looks good to me.

Reply:
@adongare Thanks Ashok, I didn't realize StyledCell also has a td. I had to merge with develop which had the new td. I'll try remove the td. Thanks!

Reply:
np

Reply:
@adongare I removed the td, thanks so much for the suggestion! Please have a look again, appreciate it!!

Reply:
looks great!